src/screens/courses/cardTemplates/OrderTheSequenceCard/index.tsx

```diff
@@ -109,7 +110,7 @@ const OrderTheSequenceCard = ({
             </>
           }
           showsVerticalScrollIndicator={false} data={answers} keyExtractor={(_, answerIndex) => answerIndex.toString()}
-          renderItem={renderItem} onDragEnd={setAnswersArray} activationDistance={10} />
+          renderItem={renderItem} onDragEnd={setAnswersArray} activationDistance={SWIPE_SENSIBILITY + 2} />
```

Comment:
a quoi correspond le +2 ?

Reply:
Il permet de faire prevaloir le swipe quand il est activé. Je ne suis pas sure a 100% mais je pense qu'`activationDistance` est un peu similaire a `activeOffsetX` sauf qu'on ne peut pas choisir horizontal/vertical. En augmentant `activationDistance` j'avais l'impression d'avoir moins de conflit entre le swipe et le scroll.

Reply:
cf pr de Kenny precedente sur le swipe: 
activationDistance = Distance a finger must travel before the gesture handler activates. Useful when using a draggable list within a TabNavigator so that the list does not capture navigator gestures.